src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,105 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 public class HelloWorldTest {
 
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        // Clear storage before each test
+        core.basesyntax.db.Storage.people.clear();
+    }
```

Comment:
Consider using `@BeforeAll` with a static setup method instead of `@BeforeEach` if the `registrationService` and `storageDao` instances do not need to be recreated before each test. This can improve the performance of the test suite if the setup is expensive and the tests do not modify the state of these objects.